hexrd/fitting/fitpeak.py

```diff
@@ -58,15 +56,15 @@ def cnst_fit_obj(x, b):
     return np.ones_like(x)*b
 
 
-def cnst_fit_jac(x, b):
+def cnst_fit_jac(x, _b):
```

Comment:
This function doesn't use `_b`. I'm not sure that changing the variable name is the right choice - if we're going to change this, we should consider fixing the signature to not have a second argument at all. Though, it may affect downstream user workflows unexpectedly.

Reply:
I believe these functions are used by the optimizer, and this is the interface the optimizer uses